bindings/kitsune_runner/src/common.rs

```diff
@@ -61,26 +64,44 @@ pub fn create_chatter(
     Ok(())
 }
 
-/// Join the chatter space.
-pub fn join_chatter_space(
+/// Return chatter id.
+pub fn chatter_id(ctx: &mut AgentContext<KitsuneRunnerContext, KitsuneAgentContext>) -> String {
+    ctx.get()
+        .chatter
+        .clone()
+        .expect("chatter_id: chatter is not created")
+        .id()
+        .to_string()
+}
+
+/// Get configured number of messages per interval.
+///
+/// This number can be configured with the env var `NUM_MESSAGES`.
+/// Defaults to 3.
+pub fn number_of_messages(ctx: &mut AgentContext<KitsuneRunnerContext, KitsuneAgentContext>) -> u8 {
```

Comment:
Would slightly prefer that this was scenario-specific logic, but maybe we want to always create data at a given rate in K2 scenarios? I guess that's all we can really do anyway :D

Reply:
Okay, I'll move this to just the scenario.

Reply:
https://github.com/holochain/wind-tunnel/pull/153/commits/b9e7882a5918c439791e7a9ab156f65c35b10cf0